### PR TITLE
Add anaconda.org upload to release

### PR DESCRIPTION
**Pull request by folmos-at-orange**

### CODE REVIEW

File: .github/workflows/conda.yml [hunk 1]
```diff
@@ -97,6 +84,7 @@ jobs:
           retention-days: 7
   # Test Conda package on brand new environments
   test:
+    if: false
```
Comment (popescu-v) [inline .github/workflows/conda.yml:87]: What is the purpose of this line? Wouldn't this make the `test` job never execute?

Comment (folmos-at-orange) [reply .github/workflows/conda.yml]: I remove it (it was as just to test fast).

File: .github/workflows/conda.yml [hunk 2]
```diff
@@ -176,26 +156,50 @@ jobs:
         with:
           miniconda-version: latest
           python-version: '3.11'
+      - name: Install anaconda client CLI
+        run: conda install anaconda-client
       - name: Reindex the package
         run: |
           conda install -y conda-build=3.27.0
           conda-index ./khiops-conda
+      - name: Extract Khiops Package Version
```
Comment (popescu-v) [inline .github/workflows/conda.yml:165]: I would put this stage just before the "Create the release zip archive", which needs the output of this stage; Anaconda upload does not need this stage AFAIU.

**Review by popescu-v — CHANGES_REQUESTED** [review]
Almost LGTM (one small caveat - see the comment).

File: .github/workflows/conda.yml (hunk 2 above)
Comment (folmos-at-orange) [reply .github/workflows/conda.yml:165]: Ok

Comment (folmos-at-orange) [reply .github/workflows/conda.yml]: Done

**Review by popescu-v — APPROVED** [review]
LGTM. I would only add a small section to CONTRIBUTING.md instructing not to reuse the same (a, b, rc, ...) tag for different commits, as this would automatically overwrite the Conda packages on the `khiops-dev`.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -241,16 +241,25 @@ For development dependencies (eg. `black`, `isort`, `sphinx`, `wrapt`, `furo`) w
...
-We follow a non-standard `MAJOR.MINOR.PATCH.INCREMENT[-PRE_RELEASE]` versioning convention. The
+We follow a non-standard `MAJOR.MINOR.PATCH.INCREMENT[PRE_RELEASE]` versioning convention. The
 first three numbers `MAJOR.MINOR.PATCH` are the latest Khiops version that is compatible with the
 package. The number `INCREMENT` indicates the evolution of `khiops-python` followed by an optional
-`[-PRE_RELEASE` version for alpha, beta and release candidate releases (eg. `-b2`).
+`[PRE_RELEASE` version for alpha, beta and release candidate releases (eg. `b2`).
 
 ## Releases
+When tagging a revision the CI will create the packages and upload them to the `khiops-dev` channel.
+Prefer to augment the pre-release revision number to re-create a tag because the CI overwrite
```
Comment (popescu-v) [inline CONTRIBUTING.md:251]: s/"because the CI overwrite"/"because otherwise the CI would overwrite"/

**Review by popescu-v — APPROVED** [review]
LGTM; can be merged IMHO

**Pull request merged**